build.gradle.kts

```diff
@@ -415,3 +416,217 @@ tasks.test.configure()
     description = "Run JUnit tests"
     dependsOn("test-core","test-plugins")
 }
+
+tasks.javadoc {
+    //TODO This function is untested because the equivalent ant target does not work
```

Comment:
Please remove this comment this is not true. Javadoc's are produced in `build/docs/api` if you run `ant javadoc`.